src/content/blog/my-attempt-at-linq-pagination.md

```
@@ -0,0 +1,60 @@
+summary: Lorem
+---
+
+<p>Today I’ve been diving into an easy way to paginate record sets in my applications.&#160; Searching around the internet yielded several good walkthroughs on how to do this.&#160; My favorite way came from my friend, Kevin Hazzardh, He discussed using the Skip() and Take() extension methods to form fit LINQ into building a SQL statement that’ll do all the heavy lifting for you.</p>  <p>Copying from Kevin’s example, I built the following code snippet:</p>
+
+```csharp
+int pageNumber = 1;
+int pageSize = 20;
+
+using (var entity = new Entities())
+{
+	var recordSet = (from r in entity.SomeTable
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)